helix-term/src/ui/editor.rs

```diff
@@ -810,6 +814,29 @@ impl Component for EditorView {
                 status_msg,
                 style,
             );
+        } else {
+            let mut disp = String::new();
+            if let Some(count) = cx.editor.count {
+                disp.push_str(&count.get().to_string())
+            }
+            if let Some(keys) = self.keymaps.pending() {
+                for key in keys {
+                    let s = key.to_string();
+                    if s.width() > 1 {
+                        disp.push_str(&format!("<{}>", s));
+                    } else {
+                        disp.push_str(&s);
+                    }
+                }
+            }
+            let width = 10usize;
```

Comment:
```suggestion
            let width = 15usize;
```
I think we can just increase this a bit?

Reply:
Done.